src/types.rs

```diff
@@ -205,3 +206,75 @@ impl<'de> serde::Deserialize<'de> for Guid {
         Guid::from_str(&s).map_err(serde::de::Error::custom)
     }
 }
+
+/// Map types
+pub mod map {
+    use std::{fmt::Debug, hash::Hash};
+
+    use indexmap::IndexMap;
+
+    /// Wrapper around `IndexMap` to implement Hash and Eq functionality.
+    #[derive(Debug, Clone, PartialEq, Eq)]
+    #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+    pub struct HashableIndexMap<K: Hash + Eq, V: Hash>(pub IndexMap<K, V>);
```

Comment:
Could it be better to also implement `From<IndexMap>` for HashableIndexMap so we could have easier conversions?

Reply:
This is already implemented in line 234-242 below: https://github.com/localcc/gvas/pull/102/files/c065c7bb1ba409f68c72155a405dd2128b975e71#diff-ed12f5ea605f23bb4d26cc65778e3daff9db3eec40e2abfc82beafca130a99a0R234-R242